### ISSUE TITLE
Comments on "Modeling reinforcement learning (Part II): Maximum likelihood estimation"

### ISSUE THREAD

**Issue by mspeekenbrink** [speekenbrink-lab/speekenbrink-lab.github.io#5]

**Comment by mirhnius** [comment]
This is a brilliant series. I really regret it not continued. It is truly insightful.

**Comment by mspeekenbrink** [comment]
Thank you for the positive feedback. it is good to hear that these posts are useful (and even insightful). I am currently writing the next one, which should be available soon.

**Comment by mirhnius** [comment]
Thanks! I'm looking forward to reading it!

**Comment by nastia-lado** [comment]
Thanks a lot for the post! It's really helpful. However, I have a question. In the section "Constraining and transforming parameters"  you transform only variance to the logarithmic scale to ensure that they are positive in unconstrained optimization can be used. As far as I understand parameter gamma (can be also called beta or inverse temperature) for softmax is also always positive, and in your bounds, you specify the lower bound as 0. But you do not apply the log transformation to gamma to insure it's positive. Is it possible that in your code gamma is positive just by chance? Because when I run your code with my data I get negative gamma.

**Comment by mspeekenbrink** [comment]
You are correct that the gamma parameter makes most sense when positive. A negative value implies more choices for relatively low rewarding options. If you'd like to constrain this parameter to always be positive, you can use a similar transformation as for the variance. This should be straightforward and ensure your estimate is positive.
